src/cmake/Setup3rdParty.cmake

```diff
@@ -151,4 +153,25 @@ if(H5ZZFP_DIR)
     endif()
 endif()
 
+##################################
+# Export BLT Targets when needed
+##################################
+
+# cmake < 3.15, we use BLT's mpi target and need to export
+# it for use downstream
+if( ${CMAKE_VERSION} VERSION_LESS "3.15.0" )
+    blt_list_append(TO BLT_TPL_DEPS_EXPORTS ELEMENTS mpi IF ENABLE_MPI)
+endif()
+
+foreach(dep ${CONDUIT_BLT_TPL_DEPS_EXPORTS})
```

Comment:
I think there's a typo here:
```suggestion
# cmake < 3.15, we use BLT's mpi target and need to export
# it for use downstream
if( ${CMAKE_VERSION} VERSION_LESS "3.15.0" )
    blt_list_append(TO CONDUIT_BLT_TPL_DEPS_EXPORTS ELEMENTS mpi IF ENABLE_MPI)
endif()

foreach(dep ${CONDUIT_BLT_TPL_DEPS_EXPORTS})
```

Reply:
Thanks, yes this was a mistake. I pushed a fix.